SD_Card_Driver/Core/Src/LOS_D_sd.cpp

```diff
@@ -0,0 +1,96 @@
+#include "LOS_D_sd.hpp"
+
+int SDCard::init()
+{
+	return f_mount(&fs, "", 1);

```

Comment:
What is the "" for?

Reply:
This string specifies the logical drive that FatFS works with. An empty string specifies the default drive.